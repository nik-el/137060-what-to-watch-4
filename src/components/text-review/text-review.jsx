import React from "react";

export const TextReview = React.memo(function TextReview() {
  return <div
    className="add-review__text">
    <textarea
      className="add-review__textarea"
      name="review-text"
      id="review-text"
      placeholder="Review text"
    />
  </div>;
});
