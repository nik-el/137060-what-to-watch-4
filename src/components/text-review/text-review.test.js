import React from 'react';
import renderer from 'react-test-renderer';

import {TextReview} from './text-review';

it(`TextReview renders correctly`, () => {
  const tree = renderer
    .create(
        <TextReview />
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});

