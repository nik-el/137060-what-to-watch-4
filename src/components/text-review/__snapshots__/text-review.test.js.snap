// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TextReview renders correctly 1`] = `
<div
  className="add-review__text"
>
  <textarea
    className="add-review__textarea"
    id="review-text"
    name="review-text"
    placeholder="Review text"
  />
</div>
`;
